nix/services.nix

```
@@ -59,7 +60,7 @@
       text = ''
         set -e
         if [[ -f evap/localsettings.py ]]; then
-            echo "Found evap/localsettings.py, exiting"
+            echo "Found evap/localsettings.py, exiting. Use clean-setup in a nix develop shell to trigger rerun."
```

Sorry forgot the other one:
```suggestion
            echo "Found evap/localsettings.py, exiting."
            echo "If you want to install a fresh environment, run clean-setup in a nix develop shell."
```